app/controllers/scim_rails/scim_users_controller.rb

```diff
@@ -126,9 +126,23 @@ def put_active_param
     end
 
     def patch_active_param
-      active = params.dig("Operations", 0, "value", "active")
-      raise ScimRails::ExceptionHandler::UnsupportedPatchRequest if active.nil?
-      active
+      handle_invalid = lambda do
+        raise ScimRails::ExceptionHandler::UnsupportedPatchRequest
+      end
+
+      operations = params["Operations"] || {}
+
+      valid_operation = operations.find(handle_invalid) do |operation|
```

Comment:
The lambda passed to `find` is called when nothing is found... so in this case if a valid operation is not found it calls the handler that will raise the UnsupportedPatchRequest exception